firmware/main.c

```diff
@@ -104,6 +104,9 @@ int main(void) {
     port_init(&port_b, 2, &PORT_B, GCLK_PORT_B,
         TCC_PORT_B, DMA_PORT_B_TX, DMA_PORT_B_RX);
 
+    // TEMPORARY
+    ws2812_init(PORT_B.power);
```

Comment:
Maybe I'm misunderstanding, but this means that this driver is now "on by default"? I don't think this is a good idea.